kb/deploy-new-version-of-workflow.md

```diff
@@ -0,0 +1,60 @@
+---
+slug: deploy-new-version-of-workflow
+title: Use Replay before deploy a new version of a Workflow
+tags:
+  - troubleshooting
+  - how-to
+date: 2023-05-11T00:00:00Z
+---
+
+When making code changes to a Workflow, it is important to avoid introducing non-deterministic behavior; otherwise, you may encounter `NonDeterministicWorkflowError` exceptions in your Workflow History.
+
+One way to prevent this exception from occurring is to use the [Workflow Replayer](/workflows#replays) to replay existing Workflow Histories against your new code before deploying it to production.
+
+If the Replay is successful, you can be confident that your new Workflow version is compatible with your existing Workflow History.
+
+## How to use the Workflow Replayer
+
+To use the Workflow Replayer, you need to provide the following:
+
+- The Workflow Replayer to register the Workflow that is going to be replayed
+- A Workflow History to execute a single Workflow Task from the JSON History file
+- (optional) Replay Workflow from a specific point in the History
+
+You can load the Event History from a JSON file on disk, obtained through one of the following methods:
+
+### Programmatically
+
+The Workflow Replayer is available in each SDK.
```

Comment:
A workflow replayer being available in each SDK seems unrelated to this section which is about loading event history programmatically